doc/contributing/PROCESS.md

```diff
@@ -168,10 +170,13 @@ To make a new OpenAssetIO release, follow this procedure.
   - If this is an alpha or beta release, ensure `Set as a pre-release`
     checkbox is checked.
   - Click "Publish Release"
-- In response to a release being published, the
-  [Deploy PyPI](https://github.com/OpenAssetIO/OpenAssetIO/actions/workflows/deploy-pypi.yml)
-  action will be invoked. Monitor this to ensure PyPI wheels are
-  uploaded correctly.
+- In response to a release being published, two things happen.
+  - The [Deploy PyPI](https://github.com/OpenAssetIO/OpenAssetIO/actions/workflows/deploy-pypi.yml)
+    action will be invoked. Monitor this to ensure PyPI wheels are
+    uploaded correctly.
+  - The [Upload Release Builds](https://github.com/OpenAssetIO/OpenAssetIO/actions/workflows/upload-release-builds.yml)
```

Comment:
[consider][minor] The job name is differently (nicely) cased
```suggestion
  - The [Upload release builds](https://github.com/OpenAssetIO/OpenAssetIO/actions/workflows/upload-release-builds.yml)
```

Reply:
fixed here : https://github.com/OpenAssetIO/OpenAssetIO/pull/1241/commits/0f2b47483cfacfde9098e37bac0d4e0fcffcafc2